recipes/gst-plugins-base/all/conanfile.py

```diff
@@ -97,12 +74,22 @@ def config_options(self):
             del self.options.with_egl
             del self.options.with_xorg
 
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+        self.options["gstreamer"].shared = self.options.shared
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
     def requirements(self):
-        self.requires("zlib/1.2.12")
-        self.requires("glib/2.72.0")
-        self.requires("gstreamer/1.19.2")
+        self.requires("glib/2.76.2", transitive_headers=True, transitive_libs=True)
+        self.requires(f"gstreamer/{self.version}", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("glib/2.76.2", transitive_headers=True)
        self.requires(f"gstreamer/{self.version}", transitive_headers=True)
```
`transitive_libs` is True by default.

Reply:
Gotcha, thank you.